app/unpack_file.py

```diff
@@ -0,0 +1,12 @@
+import json
+from pprint import pprint
+
+with open("app/config.json", "r") as file:
+    date = json.load(file)
+    fuel_price = date["FUEL_PRICE"]
+    customer_list = date["customers"]
+    shop_list = date["shops"]
```

Comment:
Let's create for that code function that return this data, and then call it where it need